cpp/libsarmata-client/sarmata_client.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __SARMATA_CLIENT_H__
+#define __SARMATA_CLIENT_H__
+
+#include "sarmata_asr.grpc.pb.h"
+
+
+namespace techmo { namespace sarmata {
+
+struct SarmataClientConfig {
+    std::string session_id = "";        // Session ID to be passed to the service. If not specified, the service will generate a default session ID itself.
```

Comment:
Enhance comment. Session ID is the best way to match log's from client application with these on server side